### PR TITLE
DOC: Update known issues, etc

### PR DESCRIPTION
**Pull request by pllim**
### Description

Update known issues and better organize the page. Also link to it from installation doc.

Fixes https://jira.stsci.edu/browse/JDAT-1662

Rendered doc preview at https://jdaviz--780.org.readthedocs.build/en/780/index.html

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [x] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [x] Is a milestone set? Milestone is only currently required for PRs related to Imviz MVP.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: docs/known_bugs.rst [hunk 1]
```diff
@@ -19,82 +36,90 @@ Another way to get the up-to-date `scikit-image` version is::
...
+Collapse and Moment Maps: Spectral bounds do not match Region selection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 When trying to do a second collapse with the same spectral region, but with
 resized bounds: change to Region=None, resize the region, then reselect Region 1,
 the region bounds are correct. However, applying Collapse again, it errors out and
 the image viewer that contained the intial collapse goes blank.
 
 
-Cubeviz cube viewer colormap printed out
```
Comment (pllim) [inline docs/known_bugs.rst:70]: This has been fixed in glue-viz/glue-jupyter#216

File: docs/known_bugs.rst [hunk 2]
```diff
@@ -19,82 +36,82 @@ Another way to get the up-to-date `scikit-image` version is::
...
------------------------------------------------------------
+In Python 3.7.1, install fails due to terminado
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Spectral bounds are off in the plugin compared to the spectrum viewer.
+In Python 3.7.1, installing jdaviz might produce this error::
 
+    ERROR: Cannot uninstall 'terminado'. It is a distutils installed project...
 
-Ghost subsets and models in spectrum viewer
```
Comment (pllim) [inline docs/known_bugs.rst:53]: @camipacifici and @PatrickOgle , I removed mention of "ghost bug" as you requested.

File: docs/known_bugs.rst [hunk 3]
```diff
@@ -19,82 +36,82 @@ Another way to get the up-to-date `scikit-image` version is::
...
----------------
+Specviz
+-------
 
-You can report a bug in the Jdaviz GitHub issues:
+Collapse Plugin spectral bounds do not match selected region
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-https://github.com/spacetelescope/jdaviz/issues/new
+Spectral bounds are off in the plugin compared to the spectrum viewer.
```
Comment (rosteen) [inline docs/known_bugs.rst:117]:
```suggestion
Spectral bound values in the plugin are sometimes off compared to the bounds shown visually in the spectrum viewer.
```

**Review by rosteen — COMMENTED** [review]
Looks good, I took the opportunity to suggest one more clarity improvement but I'm approving.

**Review by rosteen — APPROVED** [review]

**Comment by pllim** [comment]
I accepted Ricky's suggestion and POs reported they already had a look on Slack, so I am merging. Thanks!

**Pull request merged**